import catchAsyncError from "../middlewares/catchAsyncError.js";

export const getStat = catchAsyncError(async (req, res, next) => {});
